.github/actions/detect-workflow/main.go

```diff
@@ -157,6 +157,6 @@ func main() {
 	fmt.Printf("ref:%s\n", ref)
 
 	// Output of the Action.
-	fmt.Println(fmt.Sprintf(`::set-output name=repository::%s`, repository))
-	fmt.Println(fmt.Sprintf(`::set-output name=ref::%s`, ref))
+	github.SetOutput("repository", repository)
+	github.SetOutput("ref", ref)
```

Comment:
Looks like you'll need to add this on a later PR after the code gets merged since `detect-env` uses a separate `go.mod`.

Reply:
OHHH I see.